apps/explorer/lib/explorer/chain/import/runner/transactions.ex

```diff
@@ -50,6 +52,9 @@ defmodule Explorer.Chain.Import.Runner.Transactions do
     |> Multi.run(:transactions, fn repo, _ ->
       insert(repo, changes_list, insert_options)
     end)
+    |> Multi.run(:tolar_transaction_data, fn repo, _ ->
+      derive_tolar_transaction_data(repo, changes_list, insert_options)
```

Comment:
Note:

This will only work if underlying data structure will include these new fields `[:gas_refunded, :network_id, :output]`

If they are not present in the structure the importing will fail and all changes will be rolled back.

Reply:
This can be disabled by removing this call, or by returning something like `{:ok, skip}` inside the `Multi.run/3` function call.